### PR TITLE
Chibuzo-Neka manager form(css)

### PR DESCRIPTION
**Pull request by Domicaren1**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Other (please specify here: <type here>)
- [ ] 
Screenshot 2023-06-19 061347
Screenshot 2023-06-19 061548

  
  
## Description of what this Pull Request does
Styling of Neka's manager form

## Related Tickets & Documents
- Closes # (add the ticket number here, like #1)
- Related Issues: (enter ticket numbers here)
  
## QA Instructions, Screenshots, Recordings
(In this section, describe how to QA/review your PR. Feel free to use screenshots and/or short videos)
  
## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [ ] No, and here is why: __explain here__

### CODE REVIEW

**Comment by ibimina** [comment]
@Domicaren1 we don't need the image upload

**Comment by Domicaren1** [comment]
> @Domicaren1 we don't need the image upload

ok

**Comment by ibimina** [comment]
okay

**Comment by ibimina** [comment]
@jjleigh @msarit  can we merge this

File: src/pages/helperFile.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Link from 'next/link';
+
+// export const helperFile = () => {
+export const imageUpload = async (file) => {
+        try {
+          const uploadpath = `thumbnails/community/3fF17YaSFLHgRgksgVU8/${file.name}`;
```
Comment (annonymousauthority) [inline src/pages/helperFile.js:6]: Why's this hard coded? `3fF17YaSFLHgRgksgVU8`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Seen

**Pull request closed without merging**